src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/DF_BaseRecalibrator.java

```diff
@@ -193,9 +276,15 @@ protected Object doWork() {
 
         return null;
     }
+*/
+    // local files on the client's disk -> PCollections
+    private void ingestInputs(final Pipeline pipeline) throws IOException {
+
+        referencePath = BRAC.referenceArguments.getReferenceFileName();
+        // TODO: support more than one input.
+        beforePath = BRAC.readArguments.readFilesNames.get(0);
```

Comment:
Since you have `readArguments` as an optional input in your argument collection, you either need to handle the case of no read inputs, or make `readArguments` a required input (probably want to do the latter!)